kafka-ui-react-app/src/components/Nav/NavConatiner.ts

```diff
@@ -1,7 +1,10 @@
 import { connect } from 'react-redux';
-import Nav from './Nav';
-import { getIsClusterListFetched, getClusterList } from 'redux/reducers/clusters/selectors';
+import {
+  getIsClusterListFetched,
+  getClusterList,
+} from 'redux/reducers/clusters/selectors';
```

Comment:
Indeed, as @workshur mentioned it is not clear why we stretch the code over several lines here, while we collapse similar imports in other places.
This comment is just to draw your attention to the inconsistency.